src/aosm/azext_aosm/deploy/pre_deploy.py

```diff
@@ -65,11 +65,6 @@ def ensure_resource_group_exists(self, resource_group_name: str) -> None:
         if not self.api_clients.resource_client.resource_groups.check_existence(
             resource_group_name
         ):
-            if isinstance(self.config, NSConfiguration):
```

Comment:
Yay, this was bugging me yesterday and meant I couldn't use the CLI for the vnet quickstart. We could now, but I'm not going to bother changing it.  Ditto the fact you've done the same for publisher